core/src/main/java/com/github/dozermapper/core/builder/model/jaxb/AllowedExceptionsDefinition.java

```diff
@@ -88,4 +81,16 @@ public List<Class<RuntimeException>> build(BeanContainer beanContainer) {
     public ConfigurationDefinition end() {
         return parent;
     }
+
```

Comment:
Unfortunately, lombok has stopped working on the new version of jaxb2-maven-plugin. The reason is unclear. 
may be related to https://github.com/mojohaus/jaxb2-maven-plugin/issues/181